site/app/templates/forum/CreatePost.twig

```diff
@@ -93,7 +93,11 @@
         {% endif %}
 
         <span class="last-edit">
-			<strong class="post_user_id">{{ visible_username }}</strong>
+            {% if '[OP]' in visible_username %}
```

Comment:
Is there a reason to add the title attribute here?

Reply:
Yes, it is so that if users did not know what "OP" meant, then they can just hover over and see that the person is the Original Poster of the thread.